redis/src/cluster_async/mod.rs

```diff
@@ -562,36 +563,37 @@ where
             })
             .buffer_unordered(initial_nodes.len())
             .fold(
-                (HashMap::with_capacity(initial_nodes.len()), None),
+                (
+                    ConnectionsMap(HashMap::with_capacity(initial_nodes.len())),
+                    None,
+                ),
                 |mut connections: (ConnectionMap<C>, Option<String>), addr_conn_res| async move {
                     match addr_conn_res {
                         Ok((addr, node)) => {
-                            connections.0.insert(addr.into(), node);
+                            connections.0 .0.insert(addr.into(), node);
                             (connections.0, None)
                         }
                         Err(e) => (connections.0, Some(e.to_string())),
                     }
                 },
             )
             .await;
-        if connections.0.is_empty() {
+        if connections.0 .0.is_empty() {
             return Err(RedisError::from((
                 ErrorKind::IoError,
                 "Failed to create initial connections",
                 connections.1.unwrap_or("".to_string()),
             )));
         }
+        info!("create_initial_connections found nodes:\n{}", connections.0);
```

Comment:
If it's in info logs, I think it shouldn't have the function name, something more general to the user, e.g.: "Found initial nodes {connections.0:?}"